docs/content/Gameplay/mechanics/tools/Gadgets/Prospector.md

```diff
@@ -1,3 +1,21 @@
 ---
 title: Prospector
 ---
+# The Prospector
+
+The prospector is a tool used to see ores or fluids in range of you, it comes in 3 tiers with increasing sizes and capability.
+
+The Lv prospector can see a 3x3 chunk radius, and can only see ores
+
+The Hv prospector can see a 5x5 and unlocks the ability to see fluids, as well as bedrock ores when enabled in the config, and they're registered with KubeJS or datapack, see modpack dev pages.
+
+The LuV prospector can see a 7x7 radius and has the same abilities as Hv.
+
+When opening the prospector you will see ore blocks in range or fluid chunks, also if you have Xaeros map or Journeymap they will be added to your visual prospecting cache to see on the fullscreen map.
+
+On the fullscreen map you can click the fluid or ore button to switch between displays or disabling them.
+
+with FTB Chunks and Journeymap there maybe a button overlap with the display buttons, which you can fix by moving your buttons from the right to the top bar in the GregTech config.
```

Comment:
Should the default be changed, instead of instructing players how to work around it?

Reply:
Probably!